data/korath/korath outfits.txt

```diff
@@ -178,6 +189,221 @@ outfit "Control Transceiver"
 	description "This is a miniaturized communication device, which can be connected to all the major systems of a Kor Sestor automaton, apparently as a way of controlling them remotely. The communications are encrypted and authenticated in a way that would make it nearly impossible to spoof the control signals."
 
 
+outfit "Afterburner (Asteroid Class)"
```

Comment:
I just have a problem balance wise, this is way too weak in my opinion.
Okay this may provide ramscoop, and be efficient but still just compare these stats to the 10 sized human afterburner, which thrusts 5 times as much and is only 2.5 times less efficient...
I'm not even speaking about the remnant ones for which the smallest size 7 is better than the size 55 version of this thing (it uses 15 times as much fuel though but still, size 7); I know its the thing of remnants to have strong afterburners, but still I feel like this one needs to be buffed.
Either we make it even more efficient and really be an afterburner that is close to an actual engine in thrust, or we buff its thrust.
But when it is said this nearly explodes my engines I expect this thing to be at least 2x as strong than what it is currently.